src/wallet/rpcwallet.cpp

```diff
@@ -2409,6 +2409,7 @@ static UniValue getbalances(const JSONRPCRequest& request)
             "      \"trusted\": xxx                 (numeric) trusted balance (outputs created by the wallet or confirmed outputs)\n"
             "      \"untrusted_pending\": xxx       (numeric) untrusted pending balance (outputs created by others that are in the mempool)\n"
             "      \"immature\": xxx                (numeric) balance from immature coinbase outputs\n"
+            "      \"used\": xxx                    (numeric) balance from coins sent to addresses that were previously spent from (potentially privacy violating)\n"
```

Comment:
7e7bb17352a3a74e142eae20572a163384fa1502

Only present if `avoid_reuse` flag is set.